spec/features/registration/resend_confirmation_instructions_spec.rb

```diff
@@ -3,77 +3,83 @@
 require 'support/confirmation_use_case'
 
 describe 'Resending confirmation instructions', type: :feature do
-  let(:correct_email) { 'user@gov.uk' }
+  let(:unconfirmed_email) { 'user@gov.uk' }
 
   include_context 'when sending a confirmation email'
   include_context 'when using the notifications service'
 
-  context 'when entering correct information' do
-    let(:entered_email) { correct_email }
-
+  context 'when entering an email address that has signed up but not confirmed' do
     before do
-      sign_up_for_account(email: correct_email)
+      sign_up_for_account(email: unconfirmed_email)
       visit new_user_confirmation_path
-      fill_in 'user_email', with: entered_email
+      fill_in 'user_email', with: unconfirmed_email
     end
 
     it 'resends the confirmation link' do
       expect {
         click_on 'Resend confirmation instructions'
       }.to change(ConfirmationUseCaseSpy, :confirmations_count).by(1)
     end
+
+    it 'displays generic response message to the user' do
+      click_on 'Resend confirmation instructions'
+      expect(page).to have_content 'If your email address exists in our database, you will receive an email with instructions for how to confirm your email address in a few minutes.'
```

Comment:
Maybe this?

If you have an account with us, you will receive an email with a confirmation link.